src/database/models/sample/sampleOrder.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+    const Order = sequelize.define("Order", {
```

Comment:
Model name should be lowercase

Reply:
It's sample models, they will not be here for long

Reply:
Sample should help to create more... If you put a capitalized name, you will forget to put it to lowercase later...